lua/refactoring/refactor/106.lua

```diff
@@ -9,19 +9,19 @@ local helpers = require("refactoring.helpers")
 local get_selected_local_defs = require(
     "refactoring.pipeline.get_selected_local_defs"
 )
-local Config = require("refactoring.config")
 
 local M = {}
 
 local function get_code(
+    options,
```

Comment:
we could collapse options, bufnr, and lang down to just `refactor` object.

Side note, at some point I will also be refactoring the refactor object into a class of some sort.

Reply:
I agree. We could open an issue and I can start working on that too, if you want!